tests/unit/test_docs.py

```
@@ -0,0 +1,73 @@
+        return True
+    except ImportError:
+        return False
+
+
+@pytest.mark.skipif(not _pdoc_available(), reason="pdoc not installed")
+class TestDocsBuild:
+    """Verify pdoc generates docs without errors."""
+
+    def test_docs_build_exits_zero(self, tmp_path):
+        """pdoc neuraldbg exits 0 (no errors, no missing docstrings)."""
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)